rust/src/isolation.rs

```diff
@@ -0,0 +1,52 @@
+//! APIs for multi-process isolation
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use anyhow::{anyhow, Result};
+use fn_error_context::context;
+use std::process::Command;
+
+const SELF_UNIT: &str = "rpm-ostreed.service";
+/// Run as a child process, synchronously.
+const BASE_ARGS: &[&str] = &["--pipe", "--no-ask-password", "--quiet"];
```

Comment:
Shouldn't this have `--wait` too?

Reply:
Hmm I'm pretty sure it's implied by `--pipe`, but yeah let's add it anyways.